demo/src/main/java/com/bluelinelabs/conductor/demo/controllers/CustomTransitionDemoController.java

```diff
@@ -22,46 +19,36 @@
 
 public class CustomTransitionDemoController extends BaseController {
 
-    @BindView(R.id.custom_transition_container)
-    ChangeHandlerFrameLayout frameLayout;
-    @BindView(R.id.fab)
-    ImageButton fab;
+    private static final String KEY_FAB_VISIBILITY = "CustomTransitionDemoController.fabVisibility";
+
+    @BindView(R.id.fab) View fab;
 
     @Override
     protected View inflateView(@NonNull final LayoutInflater inflater, @NonNull final ViewGroup container) {
         return inflater.inflate(R.layout.controller_custom_transition, container, false);
     }
 
     @Override
-    protected void onAttach(@NonNull final View view) {
-        super.onAttach(view);
-        //dialog is visible
-        if (childRouter().getBackstackSize() > 0) {
-            fab.setVisibility(View.INVISIBLE);
-        }
+    protected void onSaveViewState(@NonNull View view, @NonNull Bundle outState) {
+        super.onSaveViewState(view, outState);
+        outState.putInt(KEY_FAB_VISIBILITY, fab.getVisibility());
+    }
+
+    @Override
+    protected void onRestoreViewState(@NonNull View view, @NonNull Bundle savedViewState) {
+        super.onRestoreViewState(view, savedViewState);
+
+        //noinspection WrongConstant
+        fab.setVisibility(savedViewState.getInt(KEY_FAB_VISIBILITY));
     }
 
     @OnClick(R.id.fab)
     public void showDialog() {
-        childRouter()
+        getRouter()
```

Comment:
Sorry, misread your question. Do you expect the dialog to be removed on rotation?